tests/impl.cpp

```diff
@@ -1136,6 +1136,21 @@ result_t test_mm_dp_ps(const SSE2NEONTestImpl &impl, uint32_t i)
     return validateFloatEpsilon(out, r[0], r[1], r[2], r[3], 2050.0f);
 }
 
+result_t test_m_psadbw(const SSE2NEONTestImpl &impl, uint32_t i)
```

Comment:
It is not worthy to validate the straightforward macro.

Reply:
It has been removed.